src/menu/menu.scss

```diff
@@ -78,13 +99,17 @@
           fill: t(iui-icons-color-primary);
         }
       }
+
+      &:focus,
+      &:focus-visible {
+        outline-offset: -2px;
```

Comment:
use variable for 2px.

Also, `:focus, :focus-visible` breaks compatibility with older browsers. I don't think you even need focus-visible here

Reply:
Using the variable and removed focus-visible.